crates/anvil/src/eth/api.rs

```diff
@@ -1436,6 +1448,58 @@ impl EthApi {
         Ok(self.filters.uninstall_filter(id).await.is_some())
     }
 
+    /// Returns EIP-2718 encoded raw transaction
+    ///
+    /// Handler for RPC call: `debug_getRawTransaction`
+    pub async fn raw_transaction(&self, hash: B256) -> Result<Option<Bytes>> {
+        node_info!("debug_getRawTransaction");
+
+        let mut out = vec![];
+
+        match self.pool.get_transaction(hash) {
+            Some(tx) => {
+                tx.transaction.encode_2718(&mut out);
+            }
+            None => return Ok(None),
+        }
+
+        if !out.is_empty() {
+            Ok(Some(out.into()))
+        } else {
+            Ok(None)
+        }
```

Comment:
here we also need to check if the tx is mined,

similar to transaction_by_hash

Reply:
wdym "check if tx is mined"? the backend struct returns unsigned txs, and they can't be encoded till they appear in pool.

Reply:
though I can do this prolly:

```rust
match self.backend.transaction_by_hash(hash).await? {
    Some(tx) => {
        Ok(Some(self.sign_transaction(tx.into_transaction_request()).await?.into()))
    }
    None => Ok(None),
}
```